File: WebApplication1/Models/Tramite.cs
namespace WebApplication1.Models
{
    public class Tramite
    {
        public long id { get; set; }
        public string fechaRecepcion {  get; set; }
        public string observacion { get; set; }
        public Persona persona { get; set; }
        public int cantidadDias { get; set; }
        public string fechaTermino { get; set; }
        public string asunto { get; set; }
    }
}

File: src/main/java/com/example/api/models/Tramite.java
package com.example.api.models;

import jakarta.persistence.*;
import lombok.Data;

import java.time.LocalDate;

@Entity
@Table(name = "trámite")
@Data
public class Tramite {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "tramiteId")
    private Long id;

    @Column(name = "fechaRecepcion")
    private LocalDate fechaRecepcion;

    @Column(name = "Observación")
    private String observacion;

    @ManyToOne
    @JoinColumn(name = "idPersona")
    private Persona persona;

    @Column(name = "Cantidad_dias")
    private Integer cantidadDias;

    @Column(name = "fechaTermino")
    private LocalDate fechaTermino;

    private String asunto;
}
